Rocket.ChatTests/CustomFields/SelectFieldSpec.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  SelectFieldSpec.swift
+//  Rocket.ChatTests
+//
+//  Created by Vadym Brusko on 10/17/17.
+//  Copyright © 2017 Rocket.Chat. All rights reserved.
+//
+
+import XCTest
+import SwiftyJSON
+
+@testable import Rocket_Chat
+
+class SelectFieldSpec: XCTestCase {
+    func testMap() {
+        // arrange
+        let selectField = SelectField()
+        let options = ["teacher", "student"]
+        let defaultValue = "teacher"
+        let json = JSON(["options": options,
+                         "defaultValue": defaultValue])
```

Comment:
Can you just use our code style to create the dictionary? It will become something like this:

```swift
let json = JSON([
    "options": options,
    "defaultValue": defaultValue
])
```